src/Uno.Toolkit.UITest/Uno.Toolkit.UITest.csproj

```diff
@@ -10,7 +10,7 @@
 		<PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
 		<PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
 		<PackageReference Include="NUnit" Version="3.13.3" />
-		<PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
+		<PackageReference Include="NUnit3TestAdapter" Version="4.3.0-alpha-net7.4" />
```

Comment:
I'm curious, why do you need this particular version?

Reply:
Otherwise I get this exception: `System.ArgumentException: Unknown framework version 7.0`

> Exception System.TypeInitializationException, Exception thrown discovering tests in C:\src\uno.toolkit.ui\src\Uno.Toolkit.UITest\bin\Uno.Toolkit.UITest\Debug\Uno.Toolkit.UITest.dll
> The type initializer for 'NUnit.Engine.Services.RuntimeFrameworkService' threw an exception.
>    at NUnit.Engine.Services.RuntimeFrameworkService.ApplyImageData(TestPackage package)
>    at NUnit.Engine.Services.RuntimeFrameworkService.SelectRuntimeFramework(TestPackage package)
>    at NUnit.Engine.Runners.MasterTestRunner.GetEngineRunner()
>    at NUnit.Engine.Runners.MasterTestRunner.Explore(TestFilter filter)
>    at NUnit.VisualStudio.TestAdapter.NUnitEngine.NUnitEngineAdapter.Explore(TestFilter filter) in D:\repos\NUnit\nunit3-vs-adapter\src\NUnitTestAdapter\NUnitEngine\NUnitEngineAdapter.cs:line 88
>    at NUnit.VisualStudio.TestAdapter.NUnit3TestDiscoverer.DiscoverTests(IEnumerable`1 sources, IDiscoveryContext discoveryContext, IMessageLogger messageLogger, ITestCaseDiscoverySink discoverySink) in D:\repos\NUnit\nunit3-vs-adapter\src\NUnitTestAdapter\NUnit3TestDiscoverer.cs:line 82
> InnerException: System.ArgumentException: Unknown framework version 7.0
> Parameter name: version
>    at NUnit.Engine.RuntimeFramework.GetClrVersionForFramework(Version frameworkVersion)
>    at NUnit.Engine.RuntimeFramework..ctor(RuntimeType runtime, Version version, String profile)
>    at NUnit.Engine.RuntimeFramework.GetNetCoreRuntimesFromDirectoryNames(IEnumerable`1 dirNames)
>    at NUnit.Engine.RuntimeFramework.FindDotNetCoreFrameworks()
>    at NUnit.Engine.RuntimeFramework.get_AvailableFrameworks()
>    at NUnit.Engine.Services.RuntimeFrameworkService..cctor()
> NUnit Adapter 4.2.0.0: Test discovery complete
> No test is available in C:\src\uno.toolkit.ui\src\Uno.Toolkit.UITest\bin\Uno.Toolkit.UITest\Debug\Uno.Toolkit.UITest.dll. Make sure that test discoverer & executors are registered and platform & framework version settings are appropriate and try again.